src/helpers/SkeletonHelper.js

```diff
@@ -13,14 +14,15 @@ const _matrixWorldInv = /*@__PURE__*/ new Matrix4();
 
 class SkeletonHelper extends LineSegments {
 
-	constructor( object ) {
+	constructor( object, options = { axesHelperSize: 0 } ) {
```

Comment:
We do not want to use the `options` approach anymore. So please inline the new parameter:
```js
constructor( object, axesHelperSize = 0 ) {
```

Reply:
>We do not want to use the options approach anymore.

Can you please link to the thread where that was decided?

Reply:
I do not know the issue where this topic was mentioned. If you don't like and additional parameter, `setAxesHelperSize()` could also be added.

Reply:
To avoid hijacking this thread, if removing `options` is something that is going to happen, I think a separate thread with a plan of action makes sense.

Reply:
Roger, happy to make the change. (I'm not an owner, your preference is my command)

The reasoning I use is this:

- for permanent required things, separate parameters is fine
- for optional things ("options") use a trailing last parameter with all the rest of the "options"
  - The advantage of this is that then we don't have this:
    ```js
    foo(requiredValue, undefined, undefined, someValue, undefined, otherValue)
    ```

Reply:
Then let's use a setter for now. When adding color configuration to axes helper, a setter was @mrdoob's preferred option, see https://github.com/mrdoob/three.js/pull/23829#issuecomment-1086452707.

Reply:
@WestLangley I should have expressed myself more clearly in https://github.com/mrdoob/three.js/pull/28314#discussion_r1593985749. When thinking about it, we did never _explicitly_ state to not use `options` anymore. I should have said using a single parameter is more in-line with other helpers and a setter more compatible to latest changes like in `CameraHelper`.